### ISSUE TITLE
I don't see the widget in my list of widgets...

### ISSUE THREAD

**Issue by mgrider** [jtbandes/space-pics#3]
No idea if I'm doing something wrong, but I haven't found a way to add the widget to my home screen. (And I would really like to do so!) I just don't see it in the widgets menu, either by scrolling or searching.

I've tried restarting my phone, and that didn't help.

I'm all up to date on iOS (16.3.1).

Any ideas?

**Comment by jtbandes** [comment]
Hi,
Are you using the workflow described here? https://support.apple.com/en-us/HT207122

Here's what it looks like on my device:
<img src="https://user-images.githubusercontent.com/14237/225139882-d4f122f1-a269-4c55-92d6-c8ce9aaa293c.jpeg" width="250">

**Comment by mgrider** [comment]
Sorry to bother you! There must be some kind of delay between install and showing up in that widgets list, because it was there when I checked again just now.

**Comment by mgrider** [comment]
Cool project, btw!!! ❤️ ❤️ ❤️

**Comment by jtbandes** [comment]
Glad to hear it!

**Comment by jtbandes** [comment]
Thanks! Curious how you found out about this project?

**Comment by mgrider** [comment]
That same stack overflow answer on scroll views. 😂 (I looked at the other closed issues, and saw I'm not the only one! heh.) Of note, the link in your comment is broken, though I think the one in the answer is correct. I ended up using `PDFKit`'s `PDFView`, but if I end up needing to customize it, I may be back to look at your view. (Though the app I'm working on is primarily is `UIKit`, it's got one or two areas I'm "borrowing" `SwiftUI` views. Heh.)
